.gitignore

```diff
@@ -15,3 +15,5 @@ internal/util/version/branch.txt
 **/testdata/fuzz
 
 docker-compose.override.yml
+
+.idea
```

Comment:
Hmm, not sure where my comment disappeared… Please check the comment on top of this file.

Reply:
My comment was attached to commit that is no longer a part of PR after rebase and force push: https://github.com/FerretDB/FerretDB/commit/ae24259849aa2e1acbf4c840c8624a3d7d5777fd#r68208013

In general, there is no need to do rebases and force pushes – there is a button GitHub UI to merge with the base branch. Or you can just do `git merge`

Reply:
Ok, got it

Reply:
> Hmm, not sure where my comment disappeared… Please check the comment on top of this file.

I can't see that comment..